doc/samples/template_testing/sensor_monitoring.rst

```diff
@@ -58,8 +79,9 @@ Wait for the device to complete the automatic registration.
 
 .. note::
    For the full installation guidelines and the application overview, refer to the `Amazon Sidewalk Sample IoT App`_ repository.
+   Flash Nordic device with the template sensor monitoring application instead of prebuilt hex file provided from cloud application repository.
```

Comment:
```suggestion
   Flash the Nordic device with the Template sensor monitoring application instead of the prebuilt HEX file provided in the cloud application repository.
```